scenarios/fork/incomplete-markers.toml

```diff
@@ -0,0 +1,29 @@
+name = "fork-incomplete-markers"
+description = '''
+The root cause the resolver to fork over `a`, but the markers on the variant
+of `a` don't cover the entire marker space, they are missing Python 3.10.
+Later, we have a dependency this very hole, which we still need to select.
```

Comment:
Can you say what the expected versus actual result is here? Just a quick prose mention is fine. (We don't have a structured way yet of describing universal resolutions in packse scenarios.)

Reply:
Add a sentence that we're testing for `c` being present